code/API_definitions/qos-booking-and-assignment.yaml

```
@@ -1310,7 +1310,7 @@ components:
 
     XCorrelator:
       type: string
-      pattern: ^[a-zA-Z0-9-_:;.\/<>{}]{0,256}$
+      pattern: "^[a-zA-Z0-9-_:;.\/<>{}]{0,256}$"

```

The pattern `^[a-zA-Z0-9-_:;.\/<>{}]{0,256}$` should use no quotes (as in Commonalities) or single quotes, but not double quotes (with double quotes the \ will be interpreted in YAML as escape, changing the regex)

Let's align with commonalities and keep this pattern unquoted:

```suggestion
      pattern: ^[a-zA-Z0-9-_:;.\/<>{}]{0,256}$
```